README.md

```diff
@@ -4,25 +4,30 @@ Crawler to collect newspaper articles for analysis
 Requirements
 ------------
 * Docker
+* Docker-Compose
 
 And that's it. Docker will get any additional requirements and setup your development or testing environment.
 
+Running
+-------
+To run the crawler, first the initial jobs must be placed in the queue. This can be done by running the ```seed.sh``` script.
+
+After the queue is seeded the ```run.sh``` script can be run.
+
+The crawler can be monitored by visiting ```localhost:5555```.
+
 Development
 -----------
 You can run the develop script to get you started. The script will build a Docker container and run it.
-```
-./develop.sh
-```
-This will start a Docker instance and connect to a bash terminal. Any changes made in the /app folder are also reflected in the running instance, allowing quick development. The exception to this is additional requirements.
+
+The ```develop.sh``` script will start a Docker instance and connect to a bash terminal. Any changes made in the /app folder are also reflected in the running instance, allowing quick development. The exception to this is additional requirements.
 
 For requirements to be loaded you need to add them to `deploy/app/requirements.txt` (python packages), or `deployment/app/Dockerfile` (ubuntu setup). A restart will be required.
 
```

Comment:
I am still not to satisfied with the wording and it seemed to have confused you as well. 

Requirements are loaded on a build (the develop script runs a build and then starts the container)
Any changes to the code are also changed in the running development container

So you can change the code at anytime, but if you change the requirements you need to rerun the script.